src/Powercord/index.js

```diff
@@ -36,6 +36,7 @@ module.exports = class Powercord extends EventEmitter {
   }
 
   async init () {
+    
```

Comment:
ok

Reply:
Listen I left this out on purpose on the first commit, and forgot to revert it before the second commit

Reply:
It's been a long ass day and my mother in law is coming over soon, so my head is in a different place